money_bot/utils/update_middleware.py

```diff
@@ -4,22 +4,23 @@
 from money_bot.utils.models import User
 
 
-async def update_user(chat_id, **kwargs):
-    user = await User.find_one({"chat_id": chat_id})
+async def update_user(user_id, **kwargs):
+    user = await User.find_one({"user_id": user_id})
 
     if user is None:
-        new_user = User(chat_id=chat_id, **kwargs)
+        new_user = User(user_id=user_id, **kwargs)
         await new_user.commit()
     else:
         for k, v in kwargs.items():
-            setattr(user, k, v)
+            if v is not None:
```

Comment:
Why the `v` can't be None? I support it's better to have a user entry with `last_name=None` than no `last_name` at all

Reply:
`last_name` field has default value, so why not?